test/test_NNBuilder.py

```diff
@@ -0,0 +1,17 @@
+from unittest import TestCase
+import torch
+import numpy as np
+from torchagents.utilities import NNBuilder
+
+class TestNNBuilder(TestCase):
+
+    def tearDown(self):
+        pass
+
+    def setUp(self):
+        self.config = {'input_size': np.random.randint(10), 'output_size': np.random.randint(10), 'hidden_layers': [128, 128, 256], 'output_function': torch.nn.Softmax}
+        self.nn = NNBuilder.LinearSequentialBuilder(**self.config)
+
+    def test_linear_sequential_builder_output(self):
+        t = torch.rand(self.config['input_size'])
+        assert self.nn(t).shape == torch.Size([self.config['output_size']])
```

Comment:
Note that you could also do self.assertTrue(a, b).

This might be better for consistency but is minor.

Reply:
Got it